minimesos/src/main/java/com/containersol/minimesos/docker/DockerContainersUtil.java

```diff
@@ -207,6 +210,7 @@ public static void pullImage(String imageName, String imageVersion, long timeout
     }
 
     /**
+     * @param containerId TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)